docs/basic_training/intro.md

```diff
@@ -186,11 +186,11 @@ The standard output shows (line by line):
 
 !!! tip
 
-    The second process runs twice, executing in two different work directories for each input file. Therefore, in the previous example the work directory `[1a/3c54ed]` represents just one of the two directories that were processed. To print all the relevant paths to the screen, use the `-ansi-log` flag (e.g. `nextflow run hello.nf -ansi-log false`).
+    The second process runs twice, executing in two different work directories for each input file. Therefore, in the previous example the work directory `[1a/3c54ed]` represents just one of the two directories that were processed. To print all the relevant paths to the screen, use the `-ansi-log` flag (e.g., `nextflow run hello.nf -ansi-log false`).
```

Comment:
I've been thinking that this text wasn't super clear about why it only showed one. Elaborated a little more here:

```suggestion
    The second process runs twice, executing in two different work directories for each input file. The [ANSI](https://en.wikipedia.org/wiki/ANSI_escape_code) log output from Nextflow dynamically refreshes as the pipeline runs; in the previous example the work directory `[1a/3c54ed]` is the second of the two directories that were processed (overwriting the log with the first). To print all the relevant paths to the screen, disable the ANSI log output usin the `-ansi-log` flag (e.g., `nextflow run hello.nf -ansi-log false`).
```